CST-Group/cst
# Moving folders to gradle default scheme

### Why was it necessary?

Some IDEs, such as Eclipse, were having a hard time being able to configure the Gradle Project within CST because source folders were not following the Gradle Nature default scheme (src/main/java ...). To avoid such problems, we changed the src, test and resource folders to Gradle default scheme.

### How was it done?

src, test and resource folders were moved to Gradle default scheme. `build.gradle` was changed in order to not point to different custom locations anymore.

### How to test?

Run a CST project that you have pointing to the version of the commit hash `d128be17bdd838f8deee3385f316ad56c9ef5930`, like the dependency `com.github.CST-Group:cst:d128be17bdd838f8deee3385f316ad56c9ef5930` in the Gradle case, and check that everything works fine.

### Future works

No more next steps in this direction ;)
